Commit:

Danila-Peshev opened the pull request:
Added some types for service working;
Install axios;
Added movie service and create movie module;
Added data for working with the TMDB in env.example.

Danila-Peshev committed:
Commit:
Added some types for service working;
Install axios;
Added movie service and create movie module;
Added data for working with the TMDB in env.example.

Isidora25 commented on src/tmdb-movie/movies.service.ts:
```diff
@@ -0,0 +1,156 @@
+    return this.httpService
+      .get(endpoint, { params })
+      .pipe(map((response) => response.data));
+  }
```
добавь обработку ошибок, на случай, если запрос упадет

Isidora25 commented on src/tmdb-movie/types/SimpleMovie.ts:
```diff
@@ -0,0 +1,12 @@
+export type SimpleMovie = {
```
Simple не очень хорошее название, все-таки 
назови base

Danila-Peshev committed:
Commit:
Rename SimpleMovie to BaseMovie;
Added cathing errors in base request MovieService.
